bokeh/settings.py

```diff
@@ -618,6 +651,17 @@ def dev(self):
     is active.
     """)
 
+    # TODO: understand this
+    validation_exceptions = PrioritizedSetting("validation_exceptions", "BOKEH_VALIDATION_EXCEPTIONS", convert=convert_validation_exceptions, default="none", help="""
+    Settings for validation error messages.
+
+    valid values are:
+
+    - ``errors`` logs errors and warnings, no exceptions (default).
+    - ``full`` logs warnings, exception on errors.
+    - ``none`` exceptions on errors or warnings.
```

Comment:
These level descriptions seem wrong. I would expect:

* none: no exceptions raised 
* error: exception raised on errors (but not on warnings)
* full: exception raised on both errors and warnings